commons/src/main/java/org/apache/mesos/elasticsearch/common/cli/validators/CLIValidators.java

```diff
@@ -65,4 +65,18 @@ public void validate(String name, String value) throws ParameterException {
             }
         }
     }
+
+    /**
+     * Ensures that the string is not empty. Will strip spaces.
+     */
+    public static class ListOfSizeTwo implements IParameterValidator {
+        @Override
+        public void validate(String name, String value) throws ParameterException {
+
+            if (value.isEmpty() || !value.contains(",") || value.split(",").length != 2) {
```

Comment:
Is the second expression actually needed here? `split(",") == 2` ensures presence of ","